sdk/core/switcher/entry.S

```diff
@@ -180,10 +180,10 @@ compartment_switcher_entry:
 	cjr				cs0
 .Lzero_arguments_start:
 	zeroRegisters	a0, a1, a2, a3, a4, a5, t0
-	// Enable interrupts of the interrupt-disable bit is not set in flags
-	bnez			t1, .Lskip_interrupt_disable
-	csrsi			mstatus, 0x8
-.Lskip_interrupt_disable:
+	// Disable interrupts if the interrupt-disable bit is  set in flags
```

Comment:
There is an extra space there

Reply:
It's worth pointing out that doing a cross-compartment call from a context with interrupts disabled will now enable interrupts, at least while in the switcher if not at the destination. This is probably fine as I don't think code running with interrupts disabled should make cross-compartment calls, but it is a potential gotcha.